resources/dicts/events/war.json

```diff
@@ -4,7 +4,8 @@
   "trigger_events": [
     "The war with o_c has begun.",
     "Rising tensions have finally come to a breaking point, a war against o_c has been declared.",
-    "lead_name declares war against o_c."
+    "lead_name declares war against o_c.",
+    "After many moons of tensions between their clans, lead_name didn't find any perspective other than a war against o_c."
```

Comment:
confusing sentence and some odd tenses going on, I'd suggest trying to reword it